app/models/hackathon/subscription/regional.rb

```diff
@@ -3,18 +3,22 @@ module Hackathon::Subscription::Regional
 
   included do
     attribute :location_input
-    validates :location, presence: true
+    validates :location, presence: {message: ->(object, _) { "'#{object.input_or_location}' could not be found." }}
```

Comment:
```suggestion
    validates :location, presence: {message: ->(object, _) { "'#{object.input_or_location}' could not be located." }}
```

Reply:
This results in "Location 'X' could not be located" - sounds kinda funny to me 🤷🏻‍♂️

Reply:
Well with an address it's "15 Falls Road could not be located", but that was more of a nitpick so we can leave it

Reply:
Since validations on attributes have the attribute's name prefixed on the message, it unfortunately looks like
```
"Location '15 Falls Road' could not be located."
```

![image](https://github.com/hackclub/hackathons-backend/assets/20099646/e1258cda-d438-4da0-95fa-cc29b1017048)